backend/src/main/resources/application.properties

```diff
@@ -10,14 +10,15 @@ quarkus.flyway.create-schemas=true
 quarkus.flyway.migrate-at-start=true
 
 quarkus.http.cors=true
-quarkus.http.cors.origins=http://localhost:3000,http://localhost:8080
+quarkus.http.cors.origins=http://localhost:5173,http://localhost:8080
 quarkus.http.cors.headers=accept, authorization, content-type, x-requested-with
 quarkus.http.cors.methods=GET, POST, OPTIONS, DELETE
 
 %dev.quarkus.devservices.enabled=true
 %dev.quarkus.hibernate-orm.log.sql=true
 %dev.quarkus.hibernate-orm.validate-in-dev-mode=true
 %dev.quarkus.flyway.schemas=quarkus
+%dev.quarkus.http.access-log.enabled=true
```

Comment:
This should go to the main branch and we need to cherry pick it on all solution branches. Same goes for the whole frontend code